charts/heimdall/README.adoc

```diff
@@ -431,16 +431,32 @@ a| `5`
 
 a| `env`
 
-Environment variables, which should be made available to the heimdall deployment. E.g.
+Environment variables, which should be made available to the heimdall deployment. Variables can be specified as key-value pairs with string values or as an object referencing a ConfigMap or Secret key. E.g.
 
 ```.yaml
 env:
   OTEL_EXPORTER_OTLP_TRACES_PROTOCOL=grpc
```

Comment:
Could you please fix these two? As you pointed out in discord, `:` should be used instead of `=`